smile_id_core/api_base.py

```diff
@@ -32,14 +32,14 @@ def __init__(self, partner_id: str, api_key: str):
 
     @classmethod
     def _make_request(
-        cls, method: str, url: str, data: dict = None, expected_status=(HTTPStatus.OK,)
+        cls, method: str, url: str, data: dict = None, expected_status=(HTTPStatus.OK,), api_version: int = 1
```

Comment:
I would suggest not passing it here, but using `self.api_version`. Reasoning - this is a private method and we don't pass arguments to it directly

Reply:
yes, sorry I missed it. you are right